package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/core-monorepo",
-  "version": "149.0.0",
+  "version": "149.0.1",
```

Hmm. Not sure that this works. Our tool always makes a major version bump, and this assumption may be baked in elsewhere as well

```suggestion
  "version": "150.0.0",
```

---

Hmm. Maybe we should move this to a new branch/PR as well? I don't recall if the branch needs to match